src/lib/duration/iso-string.js

```diff
@@ -50,3 +54,8 @@ export function toISOString() {
         (m ? m + 'M' : '') +
         (s ? s + 'S' : '');
 }
+
+export function toJSON() {
+    // this may not be fully implemented
```

Comment:
@maggiepint answered your question -- just patch `toISOString` and leave `toJSON` an alias to it.